app.rb

```diff
@@ -0,0 +1,67 @@
+require 'sinatra/base'
```

Comment:
consider splitting this into different controller files to keep each one lighter 

https://github.com/emilyalice2708/chitter-challenge/blob/master/docs/review.md#split-routes-into-separate-controller-files